deps/env-check.sh

```diff
@@ -0,0 +1,11 @@
+#/bin/sh
+
+display_usage() {
+    printf "** Please check the exported values:: **\n Daemon : $DAEMON\n Denom : $DENOM\n ChainID : $CHAINID\n DaemonHome : $DAEMON_HOME\n \n Github URL : $GH_URL\n Chain Version : $CHAIN_VERSION\n"
+    exit 1
+}
+
+if [ -z $DAEMON ] || [ -z $DENOM ] || [ -z $CHAINID ] || [ -z $DAEMON_HOME ] || [ -z $GH_URL ] || [ -z $CHAIN_VERSION ]
```

Comment:
Can we check for each env setting separately and show the message clearly. Like, `DAEMON` env setting is missing etc

Reply:
This PR is just for moving the scripts into new directories. There will be a new PR up once this gets merged in which all the script refactoring will be included

Reply:
Okay. Merging this for now. Let's address these things clearly in the follow-up PRs